### PR TITLE
Us04 crud guest

### PR DESCRIPTION
**Pull request by Merhene**
- Changed logo size in the navbar
- Added Calendar with "composer require tattali/calendar-bundle" command
- Updated CSS for calendar
- Created Guests entity
- Created CRUD for Guest entity
_________________________________
- Need to add events on the calendar

### CODE REVIEW

File: templates/guest/_calendar.html.twig [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+{% extends 'base.html.twig' %}
+
+{% block body %}
+    <div id="calendar-holder"></div>
+{% endblock %}
+
+{% block javascripts %}
+    <script src="https://cdn.jsdelivr.net/npm/fullcalendar@6.1.5/index.global.min.js" integrity="sha256-dHUNnePy81fXq4D/wfu7cPsEIP7zl6MvLb84jtZf+UY=" crossorigin="anonymous"></script>
+
+    <script type="text/javascript">
```
Comment (benrcd) [inline templates/guest/_calendar.html.twig]: This should be in a proper javascript file.

File: templates/guest/index.html.twig [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+                    <h6>{{ guest.beginAt ? guest.beginAt|date('d-m-Y H:i') : '' }}</h6>
+                    <h6>{{ guest.endAt ? guest.endAt|date('d-m-Y H:i') : '' }}</h6>
+				</p>
+                </a>
+			</div>
+		</div>
+                    <a href="{{ path('app_guest_show', {'id': guest.id}) }}">show</a>
+                    <a href="{{ path('app_guest_edit', {'id': guest.id}) }}">edit</a>
+        {% else %}
+            <tr>
```
Comment (benrcd) [inline templates/guest/index.html.twig]: Clean the "table" related tags

File: templates/guest/index.html.twig [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        		<div id="card">
+			<a href="{{ path('app_guest_show', {'id': guest.id}) }}"><img id="tattoo" src="{{ vich_uploader_asset(guest) }}" class="card-img-top" alt="{{ guest.id }}"></a>
+			<div class="card-body">
+				<p class="card-text">
+					<h6>{{ guest.city }}</h6>
+					<h6>{{ guest.shop }}</h6>
+                    <h6>{{ guest.beginAt ? guest.beginAt|date('d-m-Y H:i') : '' }}</h6>
+                    <h6>{{ guest.endAt ? guest.endAt|date('d-m-Y H:i') : '' }}</h6>
+				</p>
+                </a>
```
Comment (benrcd) [inline templates/guest/index.html.twig]: Lost "a" tag

File: src/EventSubscriber/CalendarSubscriber.php [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+    public function onCalendarSetData(CalendarEvent $calendar): void
+    {
+        $start = $calendar->getStart();
+        $end = $calendar->getEnd();
+        // $filters = $calendar->getFilters();
+
+        // Modify the query to fit to your entity and needs
+        // Change booking.beginAt by your start date property
+        $bookings = $this->guestRepository
```
Comment (benrcd) [inline src/EventSubscriber/CalendarSubscriber.php]: This query builder would be better in the repository.

**Review by benrcd — CHANGES_REQUESTED** [review]

**Review by Merhene — COMMENTED** [review]
- Resolved the block javascript in guest.index
- Resolved the tables tag
- Deleted lost <a> tag
- Moove the CalendarSubscriber in Repository

**Review by benrcd — APPROVED** [review]

**Pull request merged**